clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/appgateway/cache/AzureAppGatewayCachingAgent.groovy

```diff
@@ -114,6 +114,7 @@ class AzureAppGatewayCachingAgent extends AzureCachingAgent {
 
     def cacheResult = metricsSupport.transformData {
       if (updatedAppGateway) {
+        updatedAppGateway.accountName = accountName
```

Comment:
What is the reason for setting this here? I noticed that we don't do something similar in the other caching agents. Are they going to run into the same issue that prompted including this change here?